### PR TITLE
RO : Fix all Antena Play channels

### PR DESCRIPTION
**Pull request by BellezaEmporium**
Seen ZU TV around, had a look at the link. Fixed with workaround, and it seems one's session... works for all AntenaPlay channels. Enjoy this series of channels coming back from their graves.

### CODE REVIEW

File: channels/ro.m3u [hunk 1]
```diff
@@ -136,5 +136,5 @@ http://89.38.8.130:39443
 https://iptv-all.lanesh4d0w.codes/romania/utv
 #EXTINF:-1 tvg-id="VPTV.ro" tvg-country="RO" tvg-language="Romanian" tvg-logo="https://i.ibb.co/Rb2ff23/VPTV-RO.png" group-title="",VP TV (576p) [Offline]
 http://89.38.8.130:39437
-#EXTINF:-1 tvg-id="ZUTV.ro" tvg-country="RO" tvg-language="Romanian" tvg-logo="" group-title="Music",ZU TV (720p) [Offline]
-https://ivm.antenaplay.ro/live/zutv/playlist.m3u8
+#EXTINF:-1 tvg-id="ZUTV.ro" tvg-country="RO" tvg-language="Romanian" tvg-logo="" group-title="Music",ZU TV (720p)
+https://iptv-all.lanesh4d0w.codes/romania/zutv
```
Comment (jkemp814) [inline channels/ro.m3u:140]: Works

**Review by jkemp814 — DISMISSED** [review]

**Comment by LaneSh4d0w** [comment]
Found out AntenaPlay uses the same session tokens, if generated from one site, for all their services. Weird.

**Review by jkemp814 — DISMISSED** [review]
Yes, all play. Good audio and video

File: channels/ro.m3u [hunk 2]
```diff
@@ -44,7 +44,7 @@ http://89.38.8.130:39435
 #EXTINF:-1 tvg-id="GTV.ro" tvg-country="RO" tvg-language="Romanian" tvg-logo="https://i.ibb.co/bJVG9HX/GTV.jpg" group-title="Local",GTV (576p) [Not 24/7]
 rtmp://gtv1.arya.ro:1935/live/gtv1.flv
 #EXTINF:-1 tvg-id="HappyChannel.ro" tvg-country="RO" tvg-language="Romanian" tvg-logo="https://static.wikia.nocookie.net/logopedia/images/c/c3/Happy_Channel_%282020-present%29.png" group-title="Entertainment",Happy Channel (720p) [Offline]
-https://ivm.antenaplay.ro/live/happy/playlist.m3u8
+https://iptv-all.lanesh4d0w.codes/romania/happy-channel
```
Comment (jkemp814) [inline channels/ro.m3u:47]: Cannot GET /romania/happy-channel

File: channels/ro.m3u [hunk 3]
```diff
@@ -68,6 +68,8 @@ https://rtmp.digitalbroadcast.ro/moozhits/moozhits.m3u8
 https://rtmp.digitalbroadcast.ro/moozro/moozro.m3u8
 #EXTINF:-1 tvg-id="MusicChannelRomania.ro" tvg-country="RO" tvg-language="Romanian" tvg-logo="https://i.imgur.com/PRgvj4c.png" group-title="Music",Music Channel Romania (576p) [Offline]
 https://edge126.rcs-rds.ro/utvedge/musicchannelhq.stream/playlist.m3u8
+#EXTINF:-1 tvg-id="Mireasa.ro" tvg-country="RO" tvg-language="Romanian" tvg-logo="" group-title="",Mireasa
+https://iptv-all.lanesh4d0w.codes/romania/mireasa
```
Comment (jkemp814) [inline channels/ro.m3u:72]: Black screen

File: channels/ro.m3u (hunk 2 above)
Comment (LaneSh4d0w) [reply channels/ro.m3u:47]: Try again, mislabeled link on my side.

File: channels/ro.m3u (hunk 3 above)
Comment (LaneSh4d0w) [reply channels/ro.m3u:72]: Yup, same on AntenaPlay it seems, I smell the Not 24/7.

Comment (LaneSh4d0w) [reply channels/ro.m3u:72]: It seems it's a real-time reality show stream, that might explain why, linked to Antena 1. Certainly not 24/7.

File: channels/ro.m3u (hunk 2 above)
Comment (jkemp814) [reply channels/ro.m3u:47]: Working now

File: channels/ro.m3u (hunk 3 above)
Comment (jkemp814) [reply channels/ro.m3u:72]: Be nice to get that AntenaSport https://as.ro/

Comment (LaneSh4d0w) [reply channels/ro.m3u:74]: See last addition =)

File: channels/ro.m3u [hunk 4]
```diff
@@ -19,6 +19,8 @@ https://iptv-all.lanesh4d0w.codes/romania/ai
 http://stream1.antenaplay.ro/live/smil:AntenaMonden.smil/playlist.m3u8
 #EXTINF:-1 tvg-id="AntenaStars.ro" tvg-country="RO" tvg-language="Romanian" tvg-logo="https://i.imgur.com/UnkDhbr.png" group-title="Lifestyle",Antena Stars (720p)
 https://iptv-all.lanesh4d0w.codes/romania/astars
+#EXTINF:-1 tvg-id="AntenaSport.ro" tvg-country="RO" tvg-language="Romanian" tvg-logo="https://i.imgur.com/UnkDhbr.png" group-title="Sports",AntenaSport
+https://stream1.antenaplay.ro/dfs/farasecrete5/playlist.m3u8
```
Comment (jkemp814) [inline channels/ro.m3u:23]: Excellent!

**Review by jkemp814 — DISMISSED** [review]

**Comment by LaneSh4d0w** [comment]
Kept adaptive link for B1

**Review by Dum4G — APPROVED** [review]

**Review by jkemp814 — APPROVED** [review]

**Pull request merged**